cabal-install/Distribution/Client/Install.hs

```diff
@@ -582,7 +583,7 @@ printPlan :: Bool -- is dry run
 printPlan dryRun verbosity plan sourcePkgDb = case plan of
   []   -> return ()
   pkgs
-    | verbosity >= Verbosity.verbose -> notice verbosity $ unlines $
+    | verbosity >= Verbosity.verbose -> putStr $ unlines $
```

Comment:
It was mentioned in the commit msg... it's so that the output becomes easier to parse, as the line-wrapping done by `notice` makes it much harder to write a parser for the output. Moreover, even when looking myself at the out, I consider the line-wrapping rather distracting and making the output actually less readable...